controls/cis_rhel7.yml

```diff
@@ -1849,7 +1849,9 @@ controls:
     levels:
     - l1_server
     - l1_workstation
-    automated: no # rule missing
+    status: automated
+    rules:
+      - sshd_use_strong_kex
```

Comment:
I think that you should set the variable sshd_strong_kex to some specific value.

Reply:
Good catch. It is now updated in the last commit. Please, take a look in the commit message for more details.